services/frontend-service/Makefile

```diff
@@ -74,8 +74,10 @@ test-go: $(GO_FILES) proto
 	$(GO) test ./...
 
 test-ts: src/api/api.ts
-	yarn eslint
-	yarn test --watchAll=false
+	pnpm eslint
+	#pnpm is run on powershell. Without the extra --, it will be `pnpm test watchAll`.
```

Comment:
what do you mean "is run on powershell"?
We use bash or sh.

Reply:
Please either explain why a powershell-related bug is relevant for us, or adapt the comment.

Reply:
It was a misunderstanding, the actual reason is:
The extra '--' was added because of pnpm, Usage: pnpm test [-- <args>...]